jujugui/static/gui/src/app/views/topology/topology.js

```diff
@@ -134,6 +134,39 @@ YUI.add('juju-topology', function(Y) {
 
       vis = svg.append('svg:g');
       this.vis = vis;
+
+      // Only add the plus service block when requested, leaving it out of
+      // the bundle details view.
+      if (this.options.includePlus) {
+        // Disable max-len, in order to maintain an easy way to paste in new
+        // assets from design should they arrive.
+        /*eslint-disable max-len*/
+        vis.html('<g class="plus-service included-plus" transform="translate(0,0)">' +
```

Comment:
Agreed.  I went with huw's solution for this, but this can become an `<image>` tag once sprited.